codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -48,6 +57,7 @@ def execute(self, input):
             business_email=input.get("businessEmail"),
             terms_agreement=input.get("termsAgreement"),
             marketing_consent=input.get("marketingConsent"),
```

Comment:
Yeah this one is weird 😂 we're not using it, @RulaKhaled pinging you since you were the last to work on this, do we know what's the intended use for this?

Reply:
We used to use this to decide if user signs up for marketing consent or not, to decide if we should opt user in for emails

Reply:
im not sure if we use it anymore though

Reply:
Might be worth following up with them, not a blocker for this PR though